src/Commands/TankDrive.h

```diff
@@ -31,11 +31,15 @@ class TankDrive: public frc::Command {
 	virtual void End();
 	virtual void Interrupted();
 
-private:
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLES
+	void SetAutomaticShifting(bool automaticShifting);
+	bool IsAutoShifterEnabled();
 
+private:
+  bool mAutomaticShifting;
+  static const int kShiftUpVelocity =  1000;
+  static const int kShiftDownVelocity = 0.75 * kShiftUpVelocity;
```

Comment:
These constants (`kShiftUpVelocity` and `kShiftDownVelocity`) do not need to be in the header file. These values are private, so no client of `TankDrive` can use them. These values may as well be in the source (`.cpp`) file.

No change required. I just want to make sure that you understand that there is no requirement that these values be in the header file.